docs/internal-pipeline/features_service_messages.md

```diff
@@ -367,6 +367,107 @@ example schema with three fields mentioned before):
 }
 ```
 
+### `workload_info_images` result schema
+
+```JSON
+{
+  "version": "1.0",
+  "fields": [
+    {
+      "name": "cluster_id",
+      "type": "String"
+    },
+    {
+      "name": "image_id",
+      "type": "String"
+    },
+    {
+      "name": "layers",
+      "type": "list"
```

Comment:
it's List with uppercase L

Reply:
Oops, I copied it from the rule, probably I need to fix it too!